john-johnconlon/homework-4/madlib-with-loops/js/madlib-console.js

```diff
@@ -0,0 +1,15 @@
+let  button = document.querySelector('#create');
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+// console.log('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
+
+
+button.addEventListener('click', function(event) {
+	event.preventDefault();
+	text = document.createTextNode('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
```

Comment:
You got this! I'm not noticing a lot of jQuery in your code, and that's totally okay if vanilla JS is your preference. Definitely look into how jQuery could have streamlined this code, and try refactoring it at some point.